pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -840,6 +807,80 @@ func builderSecretVolume(bbs corev1alpha1.BuildBuilderSpec) corev1.Volume {
 	}
 }
 
+func setupBindingVolumesAndMounts(bindings []ServiceBinding) ([]corev1.Volume, []corev1.VolumeMount, error) {
+	volumes := make([]corev1.Volume, 0)
```

Comment:
This is minor but, if we are initalizing a nil slice the general pattern should be `var volumes []corev1.Volume` if we want the performance optimization of make we should use the size we know that we will have `make([]corev1.Volume, 0, len(bindings))`